src/main/java/ui/components/NavigableListView.java

```diff
@@ -45,7 +46,9 @@
     // (when nothing is selected, both should be no-ops)
     private boolean saveSelectionCalled = false;
 
-    private IntConsumer onItemSelected = i -> {};
+    public static final boolean SHOULD_LOAD_BVIEW_ISSUE = true;
+
+    private BiConsumer<Integer, Boolean> onItemSelected = (i, e) -> {};
```

Comment:
Since this is the definition site, maybe name the parameters here better, so we take the chance to explain them.